mteb/evaluation/evaluators/utils.py

```diff
@@ -189,3 +190,59 @@ def top_k_accuracy(
         logging.info("Accuracy@{}: {:.4f}".format(k, top_k_acc[f"Accuracy@{k}"]))
 
     return top_k_acc
+
+
+def get_rank_from_dict(
+    dict_of_results: dict[str, float], doc_id: str
+) -> Tuple[int, float]:
+    tuple_of_id_score = dict_of_results.items()
+    sorted_by_score = sorted(tuple_of_id_score, key=lambda x: x[1], reverse=True)
+    for i, (id, score) in enumerate(sorted_by_score):
+        if id == doc_id:
+            return i + 1, score
+
+    return len(sorted_by_score) + 1, 0
+
+
+def evaluate_change(
```

Comment:
This seems like a utility function - should it be removed?

Reply:
Ahh right I seem to have misunderstood the change (I thought it was due to your changes of the dataset, but is of course not the case -- reviewing to many PRs these days)

Reply:
So will resolve this - I agree with your point